pkg/cmd/tests/tests_run.go

```diff
@@ -318,6 +318,28 @@ func (o *RunOptions) run(ctx context.Context, streams genericclioptions.IOStream
 		if suiteConfig.ParallelTotal > 1 {
 			suiteConfig.ParallelHost = o.ParallelServerAddress
 			suiteConfig.ParallelProcess = o.ParallelShard
+
+			ginkgo.BeforeSuite(func() {
+				if len(o.ArtifactsDir) < 1 {
+					return
+				}
+
+				d := path.Join(o.ArtifactsDir, "e2e-parallel-processes-logs")
+				err := os.Mkdir(d, 0777)
+				if err != nil && !os.IsExist(err) {
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+				}
```

Comment:
Shouldn't it be created by the root process? Otherwise all parallel processes will try to create it and all that didn't succedded will start failing because of exists error.

Reply:
Hence the check for IsExist error, so that all processes other than the first one to actually create it would just go past. It could be taken out of the `BeforeSuite` and created by the root process, I just thought this was a bit cleaner, even if not the most performant. Also that's how it's handled in the existing framework: https://github.com/scylladb/scylla-operator/blob/97c831eeb5854e58bc10500aa24c9fbb3c39998a/test/e2e/framework/framework.go#L300

Reply:
oh right, I didn't spot negation, sorry. Looks good.